File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/domain/model/VerificationBundle.java
package com.lattis.ellipse.domain.model;

public class VerificationBundle {

    private String userId;

    private boolean isVerified;

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public boolean isVerified() {
        return isVerified;
    }

    public void setVerified(boolean verified) {
        isVerified = verified;
    }

    @Override
    public String toString() {
        return "VerificationBundle{" +
                "userId=" + userId +
                ", isVerified=" + isVerified +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/domain/src/main/java/com/lattis/domain/models/VerificationBundle.kt
package com.lattis.domain.models

class VerificationBundle {

    var userId: String? = null

    var isVerified: Boolean? = false

    override fun toString(): String {
        return "VerificationBundle{" +
                "userId=" + userId +
                ", isVerified=" + isVerified +
                '}'.toString()
    }
}
